src/Speckle.Sdk/Common/HashCode.cs

```diff
@@ -130,9 +127,14 @@ private static int GetHashCode<T>(IEnumerable<T> items, int startHashCode)
     }
     else
     {
-      temp = CombineHashCodes(temp, EmptyCollectionPrimeNumber);
+      temp = CombineHashCodes(temp, EMPTY_COLLECTION_PRIME_NUMBER);
     }
 
     return temp;
   }
+
+  public int ToInt32()
```

Comment:
what's this for?  the comment says don't do it 😬

Reply:
There is a code analyser that is asking for this: https://learn.microsoft.com/en-gb/dotnet/fundamentals/code-analysis/quality-rules/ca2225

Operator overloads are not the most convenient to call from other .NET languages, or pass around as delegates etc..

Reply:
And another analyser that says that `GetHashCode` functions should never throw https://learn.microsoft.com/en-gb/dotnet/fundamentals/code-analysis/quality-rules/ca1065

Hence my other related changes.
I can't see any problems with implementing the way I have done.